contrib/pr/checkout.go

```diff
@@ -21,7 +20,10 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/spf13/afero"
```

Comment:
Can you move this import down to block below that includes the xorm import?

Reply:
Ah, sound like a bug in editor, they should group xorm imports here.